assets/css/tasks/_blog-tasks.css

```diff
@@ -0,0 +1,71 @@
+/* POSTS PAGE */
+li:first-child { /* The :first-child CSS pseudo-class represents the first element among a group of sibling elements. */
+    /* margin-top; reset the value for this margin */
+    margin-top: 0%;
+}
+
+li:last-child {
+    /* margin-bottom; reset the value for this margin */
+    margin-bottom: 0%;
+}
+
+.post-box__img {
+    /* the element should occupy the half of the available width. Take into account the width of a content and a margin */
+    /* width; */
+    /* float; set element on left */
+    /* display; the element should be type of element which occupy the entire available place */
+    /* margin-right; */
+    width: 45%;
+    float:left;
+    margin-right: 10%;
```

Comment:
You have good intuition that percentate unit is good for responsive page ;)
I've never use percentage for margin's value, and that is interesting idea. What was your motivation for usign `%` unit? 
There is so may ways to apply resposive layout: with proper units or @media query.  
I usualy use `rem` for margin property values, and for  apply responsive layout (for example a margin should change size for smaller or bigger), I add condition with media query.

But in case I need to add some smalle value for example from 1 ro 10 pixels I use `px` unit.

Reply:
I didn't know what exactly meant "half of available width" etc. in px so I tried to use percentage in width and then I just did same in margins.